### PR TITLE
Win10 utorrent

### PR DESCRIPTION
**Pull request by kayone**

### CODE REVIEW

File: src/NzbDrone.Common/Http/HttpRequestBuilder.cs [hunk 1]
```diff
@@ -232,6 +234,13 @@ public virtual HttpRequestBuilder Resource(string resourceUrl)
             return this;
         }
 
+        public virtual HttpRequestBuilder KeepAlive()
```
Comment (kayone) [inline src/NzbDrone.Common/Http/HttpRequestBuilder.cs]: this feels a bit strange, I think `KeepAlive(true)` `KeepAlive(false)` would be nicer.

File: src/NzbDrone.Common/Http/HttpRequestBuilder.cs [hunk 2]
```diff
@@ -253,6 +262,19 @@ public virtual HttpRequestBuilder SetHeader(string name, string value)
...
 
+        public virtual HttpRequestBuilder AddPrefixQueryParam(string key, object value, bool replace = false)
+        {
+            if (replace)
+            {
+                QueryParams.RemoveAll(v => v.Key == key);
+                SuffixQueryParams.RemoveAll(v => v.Key == key);
+            }
+
+            QueryParams.Insert(0, new KeyValuePair<string, string>(key, value.ToString()));
```
Comment (kayone) [inline src/NzbDrone.Common/Http/HttpRequestBuilder.cs]: why do this  for prefix but keep `Suffix`in a separate list?

Comment (kayone) [inline src/NzbDrone.Common/Http/HttpRequestBuilder.cs]: does utorrent or anyone else actually care about the order of the querystring?

Comment (Taloth) [inline src/NzbDrone.Common/Http/HttpRequestBuilder.cs]: Having less important stuff at the end (apikey, etc) makes it easier to read in log files. That's it.

Having suffix separately saves me from having an integer somewhere keeping track of the insertion point.

Comment (kayone) [inline src/NzbDrone.Common/Http/HttpRequestBuilder.cs]: In that case I think we should kill it, if your want your query string in a specific order then insert them in that order.

**Pull request merged**